### DISCUSSION TITLE
Teradata JDBC: "FetchRowCount value ignored."

### DISCUSSION THREAD

**Discussion by geuss** [dbeaver/dbeaver#14938] [Q&A] [answered]
#### System information: 
- Operating system (distribution) and version: Windows 7 SP1
- DBeaver version: 21.3.0
- Additional extensions: None

#### Connection specification:
- Database name and version: Teradata Database 15.0
- Driver name:  TeraJDBC 16.20.00.06
- Do you use tunnels or proxies (SSH, SOCKS, etc)?  No

#### Describe the problem you're observing:
When I execute any "SHOW" command in sql editor ("show" in Teradata is similar to "describe" command in oracle), DBeaver opens a windows ("Output")  with this errror:

[Teradata Database] [TeraJDBC 16.20.00.06] [Error 9605] [SQLState HY000] FetchRowCount value ignored.

#### Steps to reproduce, if exist:
execute any show command (show table "my table", show procedure "my procedure", etc...)
![image](https://github.com/dbeaver/dbeaver/assets/26387091/daf695c1-a9e9-4e2a-b94e-250a6dc91e2e)

**Comment by Matvey16** [comment]
Seems like expected Teradata behavior:

```
9605   FetchRowCount value ignored. 
9605 FetchRowCount value ignored.
Explanation:
This is a warning only, not an error. FetchRowCount is applicable only for the datarows, current request did not return any datarows, so FetchRowCount value is ignored . Response buffer will accomodate as many rows as possible.

Generated By:
OPT modules.

For Whom:
End User.

Notes:
The request is still processed despite the warning.

Remedy:
This message is for information only.
```

**Comment by geuss** [comment]
Hi, Thanks for answer. The usability and the look&feel is not very good with this frame opened every time that a user use the show command. A less cumbersome warning window?

**Comment by Matvey16** [comment, marked as answer]
@geuss You can turn this option off in properties, but you will need to open output window manually to read any specific message. The output window is opened by default when server sends output message.

![image](https://user-images.githubusercontent.com/82543000/146555676-dbfc5cdb-2b30-4831-8901-8bf7af5fb8e3.png)

**Reply by geuss** [reply]
Ok! Thank You.
